wicket-core/src/main/java/org/apache/wicket/settings/FrameworkSettings.java

```diff
@@ -45,8 +45,28 @@
  */
 public class FrameworkSettings implements IEventDispatcher
 {
+	/**
+	 *  Does the standard delivery of events. Override and do nothing if you want to disable it.
```

Comment:
Since it is `private` it is not possible to `override` it

Reply:
/**
	 *  Does the standard delivery of events. Events can be disabled at application level by setting a <code>null</code>
	 *  {@link IEventDispatcher} via {@link #setDefaultEventDispatcher(IEventDispatcher)} and setting  no additional
	 *  {@link IEventDispatcher}s (via {@link #add(IEventDispatcher)}).
	 */